### DISCUSSION TITLE
Best page to learn coding

### DISCUSSION THREAD

**Discussion by SofiaSchneider-dev** [community/community#135442] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

There're many pages but which one is the better one

**Comment by SergioVargasDev** [comment, marked as answer]
Definently: https://www.w3schools.com/

**Comment by Achraftraf** [comment]
One of the best pages to learn coding is [freeCodeCamp](https://www.freecodecamp.org/). It offers a comprehensive and free curriculum that covers web development, data science, algorithms, and more. The platform provides interactive lessons, coding challenges, and projects to build your skills step by step.

**Comment by Achraftraf** [comment]
Another excellent resource for learning coding is [Codecademy](https://www.codecademy.com/). It offers interactive courses on a variety of programming languages and technologies, with a hands-on approach that helps you practice coding skills in real time.

**Comment by Larwin64** [comment]
is this the more global community seems i got get hub backward. i can chat but my repository is still unknown.

**Comment by CarlosMendez1997Col** [comment]
Hello everyone, I am learning coding at https://www.w3schools.com/, this website offers multiple tools and programming languagues.

**Comment by SrikarHasthi** [comment]
If you are into web development check out https://www.theodinproject.com/

**Comment by incri** [comment]
you can download a lot videos from : [https://mega.nz/folder/8stDDQ7J#_mPKY8XOQ4UbaUJ5rV0WBw](url)

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/135441#discussioncomment-10317226) for reference to why this discussion was closed and locked. Thanks!
